### DISCUSSION TITLE
Am I Welcome here?

### DISCUSSION THREAD

**Discussion by SirLanko** [Cyfrin/foundry-full-course-cu#2935] [General]
Hello everyone, I am just starting with the solidity course and am excited to start learning how to code. Yes! You heard me right, I don't know coding at all but I am committed to getting through this no matter how long it takes. I would be glad if I could get support from the community, encouragement along the way, and a mentor(s) will be highly appreciated. 

It's time to lock in.

**Comment by EngrPips** [comment]
Hey @SirLanko, Welcome aboard, and we wish you the best. You can always reach out here whenever you need help. This community is always willing to help.

**Comment by Ifra001** [comment]
WELCOME TO THIS AMAZING COMMUNITY!! YOU ARE GONNA HAVE A BLAST

**Reply by EngrPips** [reply]
Indeed

**Comment by LordAdhaar** [comment]
Hi SriLanko, welcome.
